R/utils.R

```diff
@@ -104,3 +104,130 @@ get_teal_bs_theme <- function() {
     bs_theme
   }
 }
+
+
+
+
+#' Toggle button properties.
```

Comment:
please move the functions to the FilteredData-utils.R. Please add tag `@keywords internal` - this does not include function in the doc index (important for private functions)